### PR TITLE
Migrate action

### PR DESCRIPTION
**Pull request by isaacplmann**
Adds 2 new workflows
- Every week on Monday night create a new PR after running nx migrate on all the recipes
- Every night run the e2e tests for all the recipes

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+      - name: Report failure
+        if: ${{ failure() && github.repository_owner == 'nrwl' && github.event_name != 'workflow_dispatch' }}
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          status: 'failure'
+          message_format: '{emoji} Workflow has {status_message} ${{ steps.e2e-run.outcome }}'
+          notification_title: '{workflow}'
+          footer: '<{run_url}|View Run> / Last commit <{commit_url}|{commit_sha}>'
+          mention_groups: '@Isaac'
```
Comment (juristr) [inline .github/workflows/e2e.yml:43]: do you want to expand this potentially? So more ppl see it? Could go to the Nx docs/content team that Craigory has set up also for reviewing nx.dev docs

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: Run e2e tests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```
Comment (juristr) [inline .github/workflows/e2e.yml:5]: e2e tests are much needed. Should we run them as part of each PR? Just like we do on Nx. They wouldn't be just useful for migration scenarios, but also to make sure PRs with new recipes that get merged have all their tests & e2e tests passing.

File: .github/workflows/migrate-recipes.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+name: Migrate Recipes
```
Comment (juristr) [inline .github/workflows/migrate-recipes.yml:1]: I didn't expect an automated workflow, but let's try this :)

**Review by juristr — APPROVED** [review]

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (isaacplmann) [reply .github/workflows/e2e.yml:43]: This is a slack message, and the docs team is defined in github.  Once I figure out the authentication piece, we can fine tune how gets notified and where the message is posted.

**Pull request merged**